### ISSUE TITLE
Is that a way to return fixed value?

### ISSUE THREAD

**Issue by zzswang** [dzdrazil/swagger-mock-api#5]
I'm not familiar with chance, haven't figure out a way to do it.

for example, I want this url field to be a fixed value.

Thanks.

``` json
        "Image": {
            "properties": {
                "id": {
                    "type": "string"
                },
                "snapshot_url": {
                    "type": "string"
                },
                "source": {
                    "type": "string"
                },
                "url": {
                    "type": "string",
           >>>>"fixed-value": "http://chancejs.com/logo.png"
                }
            }
        }
```

**Comment by dzdrazil** [comment]
Not yet, but it's a  fairly easy thing to add.  Perhaps I can get it in tomorrow.

**Comment by dzdrazil** [comment]
Fixed with commit 24ae295910d9c9607dde8876f990a63021a5d3a6
